src/app/api/pull-request-review/route.ts

```diff
@@ -19,6 +19,8 @@ export async function POST(request: Request) {
       return NextResponse.json({ error: "Unauthorized" }, { status: 403 });
     }
 
+    console.log("request body", request);
+
     const body = await request.json();
     const {
       owner,
```

Comment:
The proposed change introduces a potential security vulnerability and should be removed. Here's why:

Logging the entire request object could expose sensitive information, including headers that might contain authentication tokens or other confidential data. This practice goes against security best practices.



It's recommended to avoid logging entire request objects. If logging is necessary for debugging, only log specific, non-sensitive information after careful consideration.```suggestion
if (!authorization || authorization !== `Bearer ${process.env.API_KEY}`) {
      return NextResponse.json({ error: "Unauthorized" }, { status: 403 });
    }

    const body = await request.json();
    const {
      owner,